lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/driver/resourcemanager/ResourceEvent.java

```diff
@@ -58,4 +60,9 @@
    * @return Runtime name of the resource
    */
   String getRuntimeName();
+
+  /**
+   * @return node label expression of the resource
+   */
+  Optional<Map<String, String>> getNodeLabelExpressions();
```

Comment:
Why is this a `Map`? Please add documentation as to what key and value are in the Java Docs.